src/components/card/index.tsx

```diff
@@ -194,13 +194,13 @@ class Card extends PureComponent<PropTypes, State> {
     }
   }
 
-  get blurBgStyle() {
-    const {enableBlur} = this.props;
+  get backgroundStyle() {
+    const {enableBlur, backgroundColor = Colors.white} = this.props;
 
     if (Constants.isIOS && enableBlur) {
       return {backgroundColor: Colors.rgba(Colors.white, 0.85)};
```

Comment:
when the blur is enabled, aren't we supposed to pass the `backgroundColor` to `rgba` function as well? 
`return {backgroundColor: Colors.rgba(backgroundColor, 0.85)};`

Reply:
Completely right! Fixed ✅